config.py

```diff
@@ -1,10 +1,86 @@
-import math as maths
-
-Width = 50  # C 50 How many squares wide the board is
-Height = 30  # C 30 Ditto but with height
-Size = 20  # C 20 The size of the sides of each square (in pixels)
-Cushion = 5  # C 10 How far the board extends beyond the visible amount
-Edge = maths.ceil(Size / 7)  # C Size/15 The gap between each cell
-Background = (120, 120, 120)  # C (120,120,120) The colour of the background
+import pygame
+
 Dead = 0
 Square = 1
+
+class cell:
+    def __init__(self):
+        self.CurrentState = Square
+        self.NextState = Dead
+
+    def kill(self):
+        self.NextState = Dead
+
+    def birth(self, state):
+        self.NextState = state
+
+
+class Board:
```

Comment:
Classes shouldn't be defined here, this file is for constants only. I'd make a board.py file with that class

Reply:
done